src/main/java/Duke.java

```diff
@@ -15,55 +43,129 @@ public static void printWordsWithIndentation(String words) {
+            break;
+        case "event" :
+            executeTaskCase(input, EVENT_STARTING_INDEX,TaskType.EVENT);
+            break;
+        default :
+            printIncorrectInputMessage();
+        }
+    }
+
+    private static void printTaskMessage() {
+        printIndentationAndDivider();
+        printWordsWithIndentation(ADDED_TO_LIST);
+        printWordsWithIndentation(tasks[Task.getTotalTasks() - 1].getStatusIconAndDescription());
+        printWordsWithIndentation(notifyNumberOfTasks());
         printIndentationAndDivider();
         System.out.println();
+    }
 
-        Scanner in = new Scanner(System.in);
-        String input = in.nextLine();
-        while (!input.equals("bye")) {
-            switch(input.split(" ")[0]) {
-            case "list" :
-                printIndentationAndDivider();
-                for (int i = 0; i < Task.getTotalTasks(); i++) {
-                    String description = tasks[i].getDescription();
-                    String statusIcon = tasks[i].getStatusIcon();
-                    printWordsWithIndentation(i + 1 + "." + "[" + statusIcon + "] " + description);
-                }
-                printIndentationAndDivider();
-                System.out.println();
-                break;
-            case "done" :
-                int index = Integer.parseInt(input.split(" ")[1]) - 1;
-                tasks[index].markAsDone();
-                printIndentationAndDivider();
-                printWordsWithIndentation("You've completed the task! Well done!");
-                printWordsWithIndentation("[" + tasks[index].getStatusIcon() + "] " + tasks[index].getDescription());
-                printIndentationAndDivider();
-                break;
-            default :
-                printIndentationAndDivider();
-                printWordsWithIndentation("added: " + input);
-                printIndentationAndDivider();
-                System.out.println();
-                tasks[Task.getTotalTasks()] = new Task(input);
-                Task.setTotalTasks(Task.getTotalTasks() + 1);
-                break;
+    private static void printIncorrectInputMessage() {
+        printIndentationAndDivider();
+        printWordsWithIndentation(TYPE_SUITABLE_COMMAND_MESSAGE);
+        printIndentationAndDivider();
+        System.out.println();
+    }
+
+    private static void executeTaskCase(String input, int starting_index, TaskType type) {
+        String description = input.strip().substring(starting_index);
+        boolean isFine = addTask(description, type);
+        if(isFine) {
+            printTaskMessage();
+        }else {
+            printIncorrectInputMessage();
+        }
+    }
+
+    private static String[] parseInputForDifferentTask(String input,TaskType type) {
+        String[] parsedOutput = {};
+        switch(type) {
+        case TODO:
+            parsedOutput = new String[]{input};
+            break;
+
```

Comment:
Perhaps you could delete the empty line between each case in the switch function. It was not shown in the lectures. Also, I think it is better to keep the consistency of format (since you did not add any empty line before or after this).